osu.Game.Rulesets.Osu.Tests/TestSceneSlider.cs

```diff
@@ -379,26 +379,25 @@ protected virtual DrawableSlider CreateDrawableSlider(Slider slider) => new Draw
 
         private void onNewResult(DrawableHitObject judgedObject, JudgementResult result)
         {
-            var osuObject = judgedObject as DrawableOsuHitObject;
-            if (osuObject == null)
-                return;
-
-            OsuSpriteText text;
-            Add(text = new OsuSpriteText
+            if (judgedObject is DrawableOsuHitObject osuObject)
```

Comment:
Personally I'd rather negate, use as guard and then keep using `osuObject` exclusively.

Reply:
For consistence, fast-return seems not aggressively used in the code base.

Reply:
I have pushed for early-return multiple times in the project. I highly prefer that pattern for large code blocks from a reading/maintenance perspective - preconditions can generally be discarded when reading the following code while otherwise you have to keep them in mind.

With that in mind, I don't particularly disagree with it in the cases in this PR.

Reply:
I'm fine with early return in most cases, but still find this one a particularly hard read when negated, since the line is creating a variable for future but not indenting code (like `using` would do). I find it easy to miss that the variable is being created in the first place.